### ISSUE TITLE
Not working in python3.6 conda environment. 

### ISSUE THREAD

**Issue by alonsoir** [CyberForce/Pesidious#8]
conda create -n py36 python=3.6 ipykernel
conda activate py36
python --version
pip install pip==8.1.1
pip install -r pip_requirements/requirements.txt

basically the problem looks like gym 0.9.2 is not installed in python 3.6 pip 8.1.1 because it needs python 3.8

This is the [output](https://pastebin.com/HHjq0NLk)

**Comment by RahatMaqsood** [comment]
Hi @alonsoir Did you manage to run the code? I have been trying to run it for a few months and I am stuck, can you kindly guide me on that? I really appreciate the help at this point.

**Comment by alonsoir** [comment]
Hi, no, i remember four months ago, the git code was working, but, I updated python, some dependency, and the code doesn't work anymore.

It is a problem of how python manage the dependencies.

i will try tomorrow. The tool worked great in the past, Undoubtedly, it was able to fool the antivirus by creating a reasonable doubt, although, if you mark in the antivirus that you only trust tools that have zero percent suspicion, pesidious will not do any good.

**Comment by RahatMaqsood** [comment]
Hi there!!
I am working on testing this code and I am having problems in the mutate
section, there is a particular file that's causing the issue
"portable_import_append". There is no extension attached to it but I
searched what type it is and it said that the file is ELF based which is
Linux based executable file. How can I run it, then? Did you encounter a
similar issue? Your help will be greatly appreciated!!

**Comment by alonsoir** [comment]
Unfortunately, the software is broken because of pip dependencies, tensorflow==1.2.1 is deprecated, and the authors need to rewrite the app in order to be up and running. I have tried everything, creating a 3.6 environment, 3.9, nothing. It is broken.

**Comment by RahatMaqsood** [comment]
Actually, I have installed the python 3.6.1 version and was able to run the
classifier. The second part of the app (mutate.py) is causing problem, as
there is a file(portable_append_import) that's being called in the code and
it's 16 bit. Do you have any information about this?

**Comment by alonsoir** [comment]
No, sorry, i tested pesidious three months ago and works fine, everything. but I lost any dependency and actually is broken. @authors need to update dependencies to python3.9, in my opinion. It's too new to have to die for having decided on such old dependencies. It's like building a Ferrari 812 out of parts from a 1960s Ferrari.

**Comment by RahatMaqsood** [comment]
Hello. Can you please guide me on this if possible??  I will really
appreciate your help. This is the solution that the authors posted for this
problem but it's not working out for me.  I am using a Windows 10 setup.

**Comment by RahatMaqsood** [comment]
Yes,  I completely agree!! May I ask,  did you run the setup on a windows
based system or linux?

**Comment by alonsoir** [comment]
I ran it in kali linux 2022 at february-march.

**Comment by RahatMaqsood** [comment]
Thank you for your response!  How were you able to install the outdated/old
versions of python libraries? Are there any separate files for that?

**Comment by alonsoir** [comment]
It's the same problem that all apps have, as they are not maintained, when a dependency is updated, the app breaks and stops working. I understand that the creators of this app are on to something else and the project is a zombie. If they're not going to keep it, they should leave it read-only.
